microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -895,7 +895,7 @@ Microsoft Edge allows you to support nonprofit organizations using Microsoft Rew
 
 ![The 'Support nonprofits' icon in the address bar](./index-images/support-nonprofit.png)
  
-To control this feature, go to `edge://settings/privacy` and turn on or off the **Show opportunities to support causes and nonprofits you care about** setting.
+To control this feature, go to `edge://settings/privacy`<!--todo: check, maybe replace by GUI nav path--> and turn on or off the **Show opportunities to support causes and nonprofits you care about** setting.
```

Comment:
this should be: **edge://settings/searchServices**

 **Settings and more** (**...**) > **Settings** > **Search and services**.

Reply:
```
To control this feature, go to **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Search and services**, 
and then turn on or off the **Show opportunities to support causes and nonprofits you care about** setting.
```